public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -82,7 +89,11 @@ export const NavToolbarActions = React.memo<Props>(({ dashboard }) => {
     if (dashboard.canEditDashboard()) {
       toolbarActions.push(
         <Button
-          onClick={dashboard.onEnterEditMode}
+          onClick={() => {
+            // TODO: new interactions for scenes
+            // DashboardInteractions.toolbarEditClick();
```

Comment:
It's okay to have them here, but I'd not drive the analytics by just instrumenting everything. Instead, I'd add them based on problems and needs